### ISSUE TITLE
How to create `blockchain_course` database?

### ISSUE THREAD

**Issue by OrfeasLitos** [decrypto-org/blockchain-course#18]
I tried following the instructions found [here](https://www.postgresql.org/docs/9.1/server-start.html). Since `postgres` cannot (and should not) be run as root, using `/usr/local/pgsql/data` as data directory was not a choice, so I created a directory within `blockchain_course` named `db_data` and ran `initdb -D db_data`. But then `postgres -D db_data` failed with `FATAL:  could not create lock file "/run/postgresql/.s.PGSQL.5432.lock": Permission denied`. Can you please help?

Tl; dr: setting up the required database should be documented.

**Comment by cnasikas** [comment]
This is not an issue related to this repository. Have you tried to `sudo chown -R postgres:postgres /var/run/postgresql` ?

> Tl; dr: setting up the required database should be documented.

I think it is.

**Comment by cnasikas** [comment]
If you mean detailed instructions how to create a database in postgres I think it is out of the scope of this repository. There are plenty available online tutorials showing how to do that.

**Comment by cnasikas** [comment]
This might help

https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-18-04

**Comment by OrfeasLitos** [comment]
Found some more instructions here: https://wiki.archlinux.org/index.php/PostgreSQL
